mim/commands/install.py

```diff
@@ -401,11 +402,11 @@ def install_from_repo(repo_root: str,
     if is_user_dir:
         install_cmd.append('--user')
 
+    os.environ['MKL_SERVICE_FORCE_INTEL'] = '1'
```

Comment:
Check whether this environment variable is necessary. Usually, we do not need that for installing MMCV.

Reply:
reslove issue #12 


Reproduce the bug
```bash
MMCV_WITH_OPS=1
```
```python
import subprocess
import numpy as np
subprocess.check_call(['python', '-m', 'pip', 'install', '/home/username/codebases/mmcv'])
```